.github/issue_template/02_performanceissue.md

```diff
@@ -0,0 +1,69 @@
+---
+name: 📉 Performance Issue
+about: Report a performance issue or regression.
+title: ''
+labels: performance
+---
+
+<!-- This is a template that helps us provide quicker feedback. Please use any relevant sections and delete anything you don't need. -->
+
+### Issue Description
+<!--
+Please include a clear and concise description of the problem.
+-->
+
+### Data
+<!--
+* Please include all information you've gathered about this performance issue. This includes:
+    -Timing
+    -Measurements
+    -Call stacks
+    -Dumps
+    -etc.
+* If possible please include text as text rather than images (so it shows up in searches).
+* If applicable please include before and after measurements.
+-->
+
+### Analysis
+<!--
+* If you have an idea where the problem might lie, let us know that here.
+* Please include any pointers to code, relevant changes, or related issues you know of.
+* If you don't know, you can delete this section.
+-->
+
+### Versions & Configurations
+<!--
+* In a visual studio developer command prompt, run `msbuild -version` and paste the output here.
+* If applicable, version of the tool that invokes MSBuild (Visual Studio, dotnet CLI, etc):
+
+Post any other relevant configuration settings here.
```

Comment:
Maybe switch to a full list? I don't like just saying "Post any relevant configuration settings here" then listing out some ideas on a single line.